src/pages/Sightseeingpage/SightseeingPage.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types'
+import Card from '../../components/Card/Card'
+import locations from '../../data.json'
+import styled from 'styled-components'
+import Header from '../../components/Header/Header'
+import Button from '../../components/Button/Button'
+
+SightseeingPage.propTypes = {
+  sightseeing: PropTypes.arrayOf(
```

Comment:
sightseeing is not a PropType. you are getting it from locations which you import directly in this file

Reply:
👍